src/test/mocks/vsc/extHostedTypes.ts

```diff
@@ -2042,20 +2042,31 @@ export namespace vscMockExtHostedTypes {
 
     export class RelativePattern implements IRelativePattern {
         base: string;
+        baseUri: vscode.Uri;
         pattern: string;
 
-        constructor(base: vscode.WorkspaceFolder | string, pattern: string) {
-            if (typeof base !== 'string') {
-                if (!base || !vscUri.URI.isUri(base.uri)) {
+        constructor(base: vscode.WorkspaceFolder | vscode.Uri | string, pattern: string) {
+            if (typeof base === 'string') {
+                // String
+                this.baseUri = vscode.Uri.from({ scheme: 'file', path: base });
```

Comment:
Actually no it's more likely this line. It's actually referencing code in the vscode module, not just a type.